components/I18nFormatters.js

```diff
@@ -9,5 +9,24 @@ export const I18nSupportLink = chunks => (
   <StyledLink href="mailto:support@opencollective.com">{chunks.length || 'support@opencollective.com'}</StyledLink>
 );
 
-const I18nFormatters = { strong: I18nBold, i: I18nItalic, SupportLink: I18nSupportLink };
+export const I18nSignLink = msg => <StyledLink href="/signin">{msg}</StyledLink>;
+export const I18nTOSLink = msg => (
+  <StyledLink color="black.500" href="/tos">
+    {msg}
+  </StyledLink>
+);
+export const I18nPrivacyLink = msg => (
+  <StyledLink color="black.500" href="/privacypolicy">
+    {msg}
+  </StyledLink>
+);
+
+const I18nFormatters = {
+  strong: I18nBold,
+  i: I18nItalic,
+  SupportLink: I18nSupportLink,
+  I18nSignLink: I18nSignLink,
+  I18nTOSLink: I18nTOSLink,
+  I18nPrivacyLink: I18nPrivacyLink,
```

Comment:
Good idea to add these to the global formatters, we use them often. Let's keep it consistent with `SupportLink`.

```suggestion
  SignInLink: I18nSignLink,
  TOSLink: I18nTOSLink,
  PrivacyPolicyLink: I18nPrivacyLink,
```

Reply:
🚀